core/modules/views/tests/handlers/views_handler_field_bulk_form.test

```diff
@@ -0,0 +1,247 @@
+<?php
+
+/**
+ * @file
+ * Definition of ViewsHandlerFieldBulkFormTest.
+ */
+
+require_once BACKDROP_ROOT . '/core/modules/views/tests/views_query.test';
+
+class ViewsHandlerFieldBulkFormTest extends ViewsSqlTest {
+
```

Comment:
Sorry that I didn't see that earlier. Newly added classes are supposed to use the lightweight testing profile:

```
protected $profile = 'testing';
```
Not sure if your test needs further adaption (the testing profile is pretty empty, no out-of-the-box content types). But maybe you don't need any.

Reply:
Thanks @indigoxela. Yeah, this creates an issue as it was relying on the standard installation nodes (although the tests seem to pass, but there are parts that are completely skipped since the view has no results). I wasn't aware of the 'testing' profile. How strict do you think adherence to that rule is? I guess I could manually create two nodes....

Reply:
The solution was simple. I just created two nodes. I'll push those changes now. Tests are all passing.